manifests/npp.pp

```diff
@@ -2,8 +2,6 @@
   $user    = 'Administrator',
   $default = true,
 )  {
-  require classroom::windows
```

Comment:
should we include the new chocolatey class here, or just let the catalog fail if the provider doesn't exist yet?